pkg/proxy/ipvs/proxier.go

```diff
@@ -378,9 +378,14 @@ func NewProxier(ipt utiliptables.Interface,
 	if kernelVersion.LessThan(version.MustParseGeneric(connReuseMinSupportedKernelVersion)) {
 		klog.ErrorS(nil, fmt.Sprintf("can't set sysctl %s, kernel version must be at least %s", sysctlConnReuse, connReuseMinSupportedKernelVersion))
 	} else {
-		// Set the connection reuse mode
-		if err := utilproxy.EnsureSysctl(sysctl, sysctlConnReuse, 0); err != nil {
-			return nil, err
+		// https://github.com/kubernetes/kubernetes/issues/93297
+		if kernelVersion.AtLeast(version.MustParseGeneric("5.9")) {
+			klog.V(2).InfoS("Left as-is", "sysctl", sysctlConnReuse)
+		} else {
+			// Set the connection reuse mode
+			if err := utilproxy.EnsureSysctl(sysctl, sysctlConnReuse, 0); err != nil {
```

Comment:
does it reads better with avoid nesting ifs?
```
if kernelVersion.LessThan(version.MustParseGeneric(connReuseMinSupportedKernelVersion)) {
   ...
} else if kernelVersion.AtLeast(version.MustParseGeneric("5.9")) {
   ...
} else {
  ...
}
```

Reply:
Thanks. Updated